Add new reference module for expiring RHEL support (#2617)
theforeman/foreman-documentation: guides/common/modules/ref_expiring-rhel-client-support-in-rhel-lifecycle-status.adoc

Reviewer: I'd probably change this to concept file I move procedure to separate proc file.

[id="expiring_rhel_client_support_in_rhel_lifecycle_status_{context}"]
= Expiring {EL} Client Support in RHEL Lifecycle Status

> {Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {EL} clients.

Reviewer: Is the use of {EL} correct in this module? @ekohl @maximiliankolb

> When either the end of maintenance support or the end of extended lifecycle support approaches in a year, you will see a notification banner in the {ProjectWebUI} if you have hosts in that {EL} version.

Reviewer: ```suggestion
When either the end of maintenance support or the end of extended lifecycle support approaches in a year, you will see a notification banner in the {ProjectWebUI} if you have hosts with that {EL} version.
```

> The notification message provides information about the {EL} version, the number of hosts running that version in your environment, the lifecycle support, and the expiration date.

Reviewer: ```suggestion
The notification provides information about the {EL} version, the number of hosts running that version in your environment, the lifecycle support, and the expiration date.
```

Along with other information, the {EL} lifecycle column is visible in the report.

> For any hosts that are not running {EL}, *Unknown* will appear in the *Lifecycle Status* and *Report* column.

Reviewer: ```suggestion
For any hosts that are not running {EL}, *Unknown* displays in the *RHEL Lifecycle status* and *Last report* columns.
```
Please double check this suggestion with SME but those are the names of columns that I see on my testing instance.

> . In the {Project} web UI, navigate to *Hosts* > *All Hosts*.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
```

. Click *Manage columns*.

> . Select the  *Content* column to expand.

Reviewer: ```suggestion
. Select the  *Content* column to expand it.
```

. Select *RHEL Lifecycle status*.

> . Click *Save* to generate a new column that displays the {EL} lifecycle status with an icon.

Reviewer: ```suggestion
. Click *Save* to generate a new column that displays the {EL} lifecycle status.
```

> You can also use the *Search* field to search hosts by `rhel_lifecycle_status`, `full_support`, `maintenance_support`, and `approaching_end_of_maintenance`.

Reviewer: ```suggestion
You can also use the *Search* field to search hosts by `rhel_lifecycle_status`.
It can have the following values:
* `full_support`
* ...
```
These are the values I have on my testing instance, you can list them all:
![image](https://github.com/theforeman/foreman-documentation/assets/108661422/a6ccfe26-1f6a-4617-8d03-b2f2216f0893)

> You can also find the {EL} lifecycle status on the *Host status* card on the host details page.
> 
> You will see an alert on the Host page for each host that runs {EL} with an upcoming End of Support (EOS) event in a year.

Reviewer: ```suggestion
You can also find the {EL} lifecycle status on the *Host status* card on the host details page.
You will see an alert on the host page for each host that runs {EL} with an upcoming End of Support (EOS) event in a year.
```

> You will see an alert on the Host page for each host that runs {EL} with an upcoming End of Support (EOS) event in a year.

Reviewer: What about if the support already ended. Will there be alert for that too?
